includes/blocks/sophi-page-list-item/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Sophi post item block
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { registerBlockType } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import save from './save';
+import block from './block.json';
+
+// CSS file.
+import './index.css';
+
+/**
+ * Register block
+ */
+registerBlockType(block.name, {
+	title: __('Sophi Recommended Post', 'sophi-wp'),
+	description: __(
+		'This is the recommended post from Sophi Site Automation for the respective page and widget combination for the parent Site Automation block.',
+		'sophi-wp',
+	),
+	parent: ['sophi/site-automation-block'],
+	edit,
+	save,
+	icon: { src: 'admin-post' },
```

Comment:
Lets use the icon from the Core List Item block here:

https://github.com/WordPress/gutenberg/blob/ba01e5d7851d6211d020687b5ae927f47ea4683e/packages/block-library/src/list-item/index.js#L4